composer.json

```diff
@@ -29,5 +29,10 @@
 		{
 			"name": "Elementor team"
 		}
-	]
+	],
+	"config": {
```

Comment:
I think it needs to be moved to another PR @matipojo

Reply:
Done.